Explorer/Assets/DCL/AvatarRendering/Wearables/Components/Wearable.cs

```diff
@@ -51,7 +51,10 @@ public Wearable(StreamableLoadingResult<WearableDTO> dto)
 
         public void ResolveDTO(StreamableLoadingResult<WearableDTO> result)
         {
-            Assert.IsTrue(!WearableDTO.IsInitialized || !WearableDTO.Succeeded);
+            // This assert breaks the wearable resolving process when not on the happy path
+            // Makes sense to be able to update the dto of the wearable
+            // Assert.IsTrue(!WearableDTO.IsInitialized || !WearableDTO.Succeeded);
```

Comment:
Why we cannot update a wearable's dto?

Reply:
mmmh I think that the wearableDTO should be request only once, starting from [here](https://github.com/decentraland/unity-explorer/blob/56c3a0cf51ae5398424be1272cc8872ebcd75eed/Explorer/Assets/DCL/AvatarRendering/Wearables/Systems/ResolveWearableByPointerSystem.cs#L106). If the wearable is not in the catalog, then request the dto, but only once.

If the DTO is resolved twice, looks like we have problem in out logic. Whats the unhappy path you encountered?

Reply:
Also, why would you update a DTO once requested? I mean, they could get redeployed while using the client, but sounds more like a cornercase for me.

Reply:
Reverted the changed but added some exception handling so the flow doesnt get broken in case something odd happens